README.md

```diff
@@ -966,13 +967,14 @@ Download [PyCM.bib](http://www.pycm.ir/PYCM.bib)
 	</tr>
 </table>
 
+## Show your support
 
-## License
+### Star this repo				
```

Comment:
Not very important but there is trailing white spaces here in header.

Reply:
Fixed in https://github.com/sepandhaghighi/pycm/pull/404/commits/d4c830d436dc3001735d2888bcb78713a8ea7844